spawn_sdk/spawn_sdk/README.md

```diff
@@ -124,10 +126,7 @@ defmodule SpawnSdkExample.Actors.UnnamedActor do
 
   alias Io.Eigr.Spawn.Example.{MyState, MyBusinessMessage}
 
-  defact sum(
-           %MyBusinessMessage{value: value} = data,
-           %Context{state: state} = ctx
-         ) do
+  action "sum", fn %Context{state: state} = ctx, %MyBusinessMessage{value: value} = data ->
```

Comment:
In examples I would like the convention for writing rpc's in the protobuf specification to be used in the name of the actions, that is, Camel Case.